docs/auroLabs.md

```diff
@@ -0,0 +1,62 @@
+
+This style of work is pre-planned and means an up-front process that includes:
+
+1. Working with a designer to not only design the new element, but stress-test in multiple scenarios, consult with other designers about intent and opportunity, and prepare the design work for integration with the Auro Design System UI Kit
+1. This designer will also need to be available to work 1:1 with a software engineer during the development process
+1. This engineer and designer team will need to consult with the Auro team eo ensure that the newly proposed element meets the criteria of becoming an Auro custom element and be included into the Auro Design System
+1. Post development there will be a review and documentation process so that the design work and the developed asset can be included into the Auro doc site
+
+This process is GREAT for planned work, but there is another channel of work. Unplanned work, that when discovered, the opportunity to add the new component to the design system is clear, but the rules of engagement are completely different. Pre-planning is gone and the time for the intentional process doesn't exist. In fact, there is even less time as this opportunity is already tied to a previously agreed upon deliverable.
+
+This is where AuroLabs comes in. It's the basic idea that developers will have an opportunity to create a new component that meets their specifications, works in their timeline and meets their goals, all the while being setup for success to be included into the Auro Design System.
+
+## A different type of engagement
+
+AuroLabs has a different playbook when it comes to submitting ideas to Auro. You have an idea? Built it. You see an opportunity? Build it. Not only do you get to build it, but there is a release pipeline available as well.
+
+All custom elements built in AuroLabs will use the `@aurolabs/` npm namespace. This whole space is dedicated to experimentation and iteration. For most, developing custom elements is a new way of thinking and a new way of doing work. That means learning new things. AuroLabs allows for that learning process to happen in a way that works with everyone's timelines.
+
+## From the minors to the majors
+
+The process for releasing a new custom element from AuroLabs to the Auro Design System is something the Auro team metaphorically refer to as going from the [minors](https://auro.alaskaair.com/aurolabs/minors) to the [majors](https://auro.alaskaair.com/aurolabs/majors). It's really a process of learning, iterating, coaching, doting the i's and crossing the t's.
+
+The goal always being, the Auro team will work closely with anyone to ensure that the new added custom element is the best it can be and has a solid place in the Auro ecosystem.
```

Comment:
```suggestion
The Auro team will work closely with anyone to ensure that the new custom element is the best it can be and has a solid place in the Auro ecosystem.
```